### DISCUSSION TITLE
Patch 2.7.2

### DISCUSSION THREAD

**Discussion by APickledWalrus** [SkriptLang/Skript#6168] [General]
# Skript 2.7.2

As per the [new release model](https://github.com/SkriptLang/Skript/blob/master/CLOCKWORK_RELEASE_MODEL.md), the first of the month brings with it a new Skript release! This release includes several bug fixes for issues that have been reported.

Thank you all for your continued support.

Happy Skripting!

### Bug Fixes
- [#5765](<https://github.com/SkriptLang/Skript/pull/5765>) Fixed an error with non-finite vectors
- [#6038](<https://github.com/SkriptLang/Skript/pull/6038>) Fixed the JavaDocs name and title elements
- [#6078](<https://github.com/SkriptLang/Skript/pull/6078>) Updated documentation links to the new Minecraft wiki 
- [#6080](<https://github.com/SkriptLang/Skript/pull/6080>) Fixed issues with language nodes in command help
- [#6090](<https://github.com/SkriptLang/Skript/pull/6090>) Fixed the fake player count syntax for Paper
- [#6102](<https://github.com/SkriptLang/Skript/pull/6102>) Fixed an error when sorting lists
- [#6103](<https://github.com/SkriptLang/Skript/pull/6103>) Fixed UTF-8 encoding when building Skript
- [#6106](<https://github.com/SkriptLang/Skript/pull/6106>) Fixed reloading a directory with the script file effect
- [#6117](<https://github.com/SkriptLang/Skript/pull/6117>) Fixed an error when printing block inventories
- [#6121](<https://github.com/SkriptLang/Skript/pull/6121>) Fixed an issue where Skript options would not work in functions in some cases
- [#6126](<https://github.com/SkriptLang/Skript/pull/6126>) Fixed permission messages not showing for Skript commands
- [#6128](<https://github.com/SkriptLang/Skript/pull/6128>) Fixed issue when checking if Minecraft time is between two values that span across midnight
- [#6130](<https://github.com/SkriptLang/Skript/pull/6130>) Fixed issues with changing in the drops expression not setting all items and possibly causing an error
- [#6132](<https://github.com/SkriptLang/Skript/pull/6132>) Fixed floating point error that could occur when trying to loop with "x times"
- [#6150](<https://github.com/SkriptLang/Skript/pull/6150>) Fixed the message when reloading multiple scripts in a directory
- [#6154](<https://github.com/SkriptLang/Skript/pull/6154>) Fixed an issue with changing the durability of an item

[Click here to view the full list of commits made since 2.7.1](<https://github.com/SkriptLang/Skript/compare/2.7.1...2.7.2>)

### Help Us Test
We have an [official Discord community](https://discord.gg/ZPsZAg6ygu) for beta testing Skript's new features and releases.

### Compatibility Warning
Due to the major internal changes within the 2.7 update, some addons may no longer work properly.
Please be patient as addon developers work to update their addons.
We have published a new release of our [Addon Patcher](https://github.com/SkriptLang/AddonPatcher/releases/tag/2.7.0-dev1), but be aware that it cannot fix all issues.

### Thank You

Special thanks to the contributors whose work was included in this version:
- @3meraldK
- @TheLimeGlass
- @MihirKohli
- @Spongecade
- @UnderscoreTud
- @AyhamAl-Ali 
- @Moderocky
- @APickledWalrus 
- @sovdeeth

As always, if you encounter any issues or have some minor suggestions, please report them at https://github.com/SkriptLang/Skript/issues.
If you have any bigger ideas or input for the future of Skript, you can share those too at https://github.com/SkriptLang/Skript/discussions.

<hr /><em>This discussion was created from the release <a href='https://github.com/SkriptLang/Skript/releases/tag/2.7.2'>Patch 2.7.2</a>.</em>

**Comment by nemusyy** [comment]
thank you for fixing my issue
